n3fit/src/n3fit/backends/keras_backend/MetaLayer.py

```diff
@@ -8,8 +8,8 @@
     For instance: np_to_tensor is just a call to K.constant
 """
 
-from tensorflow.keras.layers import Layer
-from tensorflow.keras.initializers import Constant, RandomUniform, glorot_normal, glorot_uniform
+from keras.initializers import Constant, RandomUniform, glorot_normal, glorot_uniform
+from keras.layers import Layer
```

Comment:
I'm not such a fan of this. Yes, keras 3 now supports multiple backends but we're still only using tensorflow (and in various places inside `keras_backend` we explicitly use tensorflow, not just keras) so to avoid possible sources of conflict I'd rather just keep using the integrated `tf.keras`